### ISSUE TITLE
Crowd is too quiet

### ISSUE THREAD

**Issue by twhippp** [YARC-Official/YARG#854]
In official games, the crowd is the same volume as or even overpowers the song. In YARG, I can barely hear the crowd.

**Comment by EliteAsian123** [comment]
This is because YARG doesn't really have a "crowd" like in the official games, it just uses the crowd stem from the song, which a lot of songs don't have, or only have in the chorus of the song.

Some sort of crowd noise is something that is planned for the future, it's just not something that is a priority at the moment unfortunately.

**Comment by thomeval** [comment]
I'm guessing you're referring to songs that do actually have a crowd stem? I've noticed them to be noticeably quieter than official games as well. While you can lower all the other stem volumes in-game to make the crowd more noticeable, I think the issue here (at least with the songs I've tested) is that the crowd stems themselves are quieter than they should be. Not sure why this would be the case, but you could fix it by making them louder using a tool like Audacity, assuming you've got individual audio files to work with.

**Comment by twhippp** [comment]
yes I am referring to crowd audio from songs. They are much much quieter
than the song whereas if you listen to the files normally, they sound fine.
It’s as if YARG is making them quieter in the mix.

**Comment by twhippp** [comment]
> This is because YARG doesn't really have a "crowd" like in the official games, it just uses the crowd stem from the song, which a lot of songs don't have, or only have in the chorus of the song.
> 
> Some sort of crowd noise is something that is planned for the future, it's just not something that is a priority at the moment unfortunately.

My apologies, I mean songs that have a crowd stem file. I’m manually turning them up to make them loud enough in the game.

**Comment by BenMcLean** [comment]
> I'm guessing you're referring to songs that do actually have a crowd stem? I've noticed them to be noticeably quieter than official games as well. While you can lower all the other stem volumes in-game to make the crowd more noticeable, I think the issue here (at least with the songs I've tested) is that the crowd stems themselves are quieter than they should be, and YARG is playing them back correctly. Not sure why this would be the case, but you could fix it by making them louder using a tool like Audacity, assuming you've got individual audio files to work with.

I haven't played the game in a while but when I started it up today, I found that crowd sounds have been added in an "SFX" channel which I really, really don't want. But also, muting the SFX channel probably is muting other non-crowd sounds that I do want. Seems like if you're gonna add crowd sounds, they should go in their own channel? Or in the crowd channel, to only be used if there's no crowd stem?
